force-app/main/default/cspTrustedSites/googleapis.cspTrustedSite-meta.xml

```diff
@@ -3,4 +3,10 @@
     <context>All</context>
     <endpointUrl>https://www.googleapis.com</endpointUrl>
     <isActive>true</isActive>
+    <isApplicableToConnectSrc>true</isApplicableToConnectSrc>
+    <isApplicableToFontSrc>true</isApplicableToFontSrc>
```

Comment:
@rax-it please only enable the `connect-src` policy. This particular API is only ever used in the capacity of invoking `fetch`. Enabling other policies violates the principle of least access in this context.

Reply:
@rax-it, also, since master works fine in its current state (even deployed to a v49 api org), please change your PR target to the `prerelease/summer20` branch. This is where we're putting all changes that will be pushed into master for when R2 goes live in core. 

Thanks!

Reply:
@pchittum Thanks for pointing it out, I updated the file. Also as pointed out in the slack conversation as this is an issue with 228 is Winter ’21 but I don't see a branch for that yet.

Reply:
@rax-it will the metadata work in Summer 20? If so, no harm in merging now and saving us work later. :-)